### DISCUSSION TITLE
Add option to ignore displays for moving

### DISCUSSION THREAD

**Discussion by dartess** [rxhanson/Rectangle#1424] [Ideas]
I use three displays—two main ones for work. And one small one, auxiliary. When I try to move a window to another display, the window always ends up on the small screen first, and only then on the second working one. I would like to completely ignore the small display, since the set of windows on it never changes.

**Comment by rxhanson** [comment]
Thanks for the feedback! I don't have the bandwidth to add this in at the moment, but see no reason to turn down a good pull request for it.
